src/scss/includes/z_index.scss

```diff
@@ -32,4 +32,16 @@
 
 .map_control_group {
   z-index: 2;
-}
\ No newline at end of file
+}
+
+.autocomplete_suggestions {
+  z-index: 4;
+}
+
+.top_bar {
+  z-index: 2;
+}
+
+.service_panel_mobile {
+  z-index: 3;
+}
```

Comment:
There is a problem on mobile with these z-index values.  
The "favorites" button stays on top of the input field and the cross is hidden:
![image](https://user-images.githubusercontent.com/4726554/47580564-7c8c1880-d94f-11e8-9df5-fdfd7c92be28.png)